Platform/Google/msm8998/walleye.dsc

```diff
@@ -0,0 +1,28 @@
+[Defines]
+  PLATFORM_NAME                  = walleye
+  VENDOR_NAME                    = Google
+  PLATFORM_GUID                  = 28f1a3bf-193a-47e3-a7b9-5a435eaab2ee
+  PLATFORM_VERSION               = 0.1
+  DSC_SPECIFICATION              = 0x00010019
+  OUTPUT_DIRECTORY               = Build/$(PLATFORM_NAME)
+  SUPPORTED_ARCHITECTURES        = AARCH64
+  BUILD_TARGETS                  = DEBUG|RELEASE
+  SKUID_IDENTIFIER               = DEFAULT
+  FLASH_DEFINITION               = Platform/Qualcomm/msm8998/msm8998.fdf
+  DEVICE_DXE_FV_COMPONENTS       = Platform/Google/msm8998/walleye.fdf.inc
+
+!include Platform/Qualcomm/msm8998/msm8998.dsc
+
+[BuildOptions.common]
+  GCC:*_*_AARCH64_CC_FLAGS = -DENABLE_SIMPLE_INIT -DENABLE_LINUX_SIMPLE_MASS_STORAGE -DAB_SLOTS_SUPPORT
```

Comment:
Should "AB_SLOTS_SUPPORT=TURE" not be used?

Reply:
I enabled AB Slots in walleye platform

Reply:
Is there a better way to enable AB slots support ??

Reply:
yes，This improvement was added in the commit long ago